src/main/resources/application.yml

```diff
@@ -22,6 +22,12 @@ spring:
     maxActive: 20
     testOnBorrow: true
     validationQuery: SELECT 1
+
+hystrix.command.default.execution.isolation.strategy: SEMAPHORE
+hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds: 15000
+hystrix.command.default.circuitBreaker.requestVolumeThreshold: 20
+hystrix.command.default.circuitBreaker.sleepWindowInMilliseconds: 5000
```

Comment:
I thought it will be useful to have it because we can create other commands and their timeout is taken from that property it will be more clear. maybe I should remove it for now